recipe/deploy/writable.php

```diff
@@ -87,18 +82,17 @@
         // -L   traverse every symbolic link to a directory encountered
         run("$sudo chown -L $recursive $httpUser $dirs");
     } elseif ($mode === 'chgrp') {
-        try {
```

Comment:
Why this block was removed?

Reply:
Because I do not consider it really helpful and it's not common in the codebase to give hints why a command failed.

Basically no other command gives feedback when it fails, so why here?

And, the commands can fail for many reasons (lost connection, wrong group, broken server, wrongly configured sudo, …), so why assume that the problem is that the user is not part of the group? What about all the other cases where the message may be just misleading, because the problem has nothing to do with the groups of the user?

Reply:
Make sense.